testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -254,7 +255,7 @@ def test_LAMMPSDCDReader_set_dt(self, u, dt=1500.):
         assert_almost_equal(u.trajectory[iframe].time, iframe * dt,
                             err_msg="setting time step dt={0} failed: "
                                     "actually used dt={1}".format(
-                                dt, u.trajectory._ts_kwargs['dt']))
+            dt, u.trajectory._ts_kwargs['dt']))
```

Comment:
Not that it matters, but I'm not sure this doesn't make the visual indentation worse 🤔